src/app/api/v1/recordings/[id]/route.ts

```
@@ -0,0 +1,33 @@
+import { NextRequest, NextResponse } from 'next/server';
+import { APIError } from '@/global/exceptions';
+import { IRecording } from '@/global/types';
+import MESSAGES from '@/global/messages';
+import { API_ERROR_CODE } from '@/global/constants/errorCodes';
+import getReqBody from '@/app/api/api-utils/getReqBody';
+import { recordingDocRef } from '../../firestoreRefs';
+
+export const POST = withErrorHandling(async (req: NextRequest) => {
+	const { recordingId } = await getReqBody(req);
+
+	if (isNaN(recordingId)) throw new APIError(`${MESSAGES.REQ_BODY_ERROR}`, 500, API_ERROR_CODE.REQ_BODY_ERROR);
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)